packages/docs/src/content/03-components/structure/section/overview.mdx

```diff
@@ -1,31 +1,46 @@
-# Beispiel
+# Playground
+
+Verwende `<Section />`, um eine Section darzustellen. Innerhalb einer Section
```

Comment:
, um eine Section darzustellen. Eine Section erzeugt automatisch feste Abstände zwischen den Elementen, die in ihr platziert werden